website/snippets/_warehouse-volatility.md

```diff
@@ -0,0 +1,14 @@
+<Expandable alt_header="Warehouse-specific volatility keywords">
+
+Different warehouses show volatility controls with different keywords and default values:
```

Comment:
i really like this breakdown - could we do the same for `entry_point` config for python UDFs?

https://github.com/dbt-labs/docs.getdbt.com/pull/8116/files#diff-df0a408abd818d4109e01c610c4d8eede58505d7d312ca5f1e5bbd9c5775731e

Reply:
This looks correct

Reply:
added: https://github.com/dbt-labs/docs.getdbt.com/pull/8116/files